packages/algob/src/lib/msig.ts

```diff
@@ -29,24 +28,37 @@ export async function decodeMsigObj (msig: string): Promise<MultiSig> {
 
 /**
  * Description: this function reads multisig from /assets/<filename>.msig
+ *              and returns the decoded multisig object
  * @param {string} msig : multisigned msig obj
- * @returns {[string, string]} : cached .teal filename, raw msig object as string
+ * @returns {string} : raw msig object as string
```

Comment:
Wrong return parameter. It's not a string.

Reply:
Yes, updated.